lib/streamlit/elements/image.py

```diff
@@ -68,9 +68,12 @@ def image(
         width : int or None
             Image width. None means use the image width.
```

Comment:
Product side note, I feel like none should use column width but that's just me

Reply:
Oh -- I agreed, and `width=None` actually does default to `use_column_width='auto'`, aka the image will use its natural width but not exceed the column.

I've update the docstring here!

Reply:
LGTM